services/mmc/plugins/xmppmaster/master/agentmaster.py

```diff
@@ -1751,6 +1754,22 @@ def MessagesAgentFromChatroomMaster(self, msg):
                 else:
                     logger.error("** enregistration base error")
                     return
+                
+                #manage update remote agent
+                if 'finger_print_remote_agent' in data and \
+                    data['finger_print_remote_agent'].upper() != 'DEV' and \
+                    data['finger_print_remote_agent'].upper() != 'DEBUG' and \
+                    data['finger_print_remote_agent'] != self.Update_Remote_Agentlist.get_fingerprint_agent_base() and \
+                    self.Update_Remote_Agentlist.autoupdate != False:
```

Comment:
> [E712] comparison to False should be 'if cond is not False:' or 'if cond:'
> [link](https://sider.review/gh/86473768/pull_requests/234)

The line 1763 seems to be :
self.Update_Remote_Agentlist.autoupdate is not False: